### PR TITLE
Add compass metapackage

### LINKED ISSUE #283: Add COMPASS metapackage
Having a defined metapackage for the COMPASS (or at least COMPASS-Ocean) conda environment would be very helpful in keeping track of and updating the environment, including version numbers.

### PR DESCRIPTION
**Pull request by xylar**
Waiting for `jigsawpy` to be released.

closes #283

### CODE REVIEW

File: compass/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - geometric_features 0.1.5
+    - mpas_tools 0.0.6
+    - jigsaw 0.9.12
+    - jigsawpy 0.0.3
```
Comment (xylar) [inline compass/meta.yaml]: This version of `jigsawpy` is on `e3sm`, not `conda-forge` for now.  It's just a placeholder.

**Comment by xylar** [comment]
All packages are now in place.  I'm going to create test environments tomorrow and invite folks to try them out.

**Comment by xylar** [comment]
Based on issues in https://github.com/MPAS-Dev/MPAS-Model/pull/441, we're likely going to need to wait for a cartopy release.  It depends on how @sbrus89's tests go.

**Comment by sbrus89** [comment]
@xylar as I mentioned in [MPAS-Dev/MPAS-Model#441](https://github.com/MPAS-Dev/MPAS-Model/pull/441), We don't need to wait for the `cartopy 0.18.0b1` release.

**Pull request merged**